src/main/kotlin/com/openlattice/postgres/PostgresDataTables.kt

```diff
@@ -148,6 +159,9 @@ class PostgresDataTables {
                     .name(prefix+"_read_data_idx")
                     .ifNotExists()
 
+            val needsPropagateIndex = PostgresExpressionIndexDefinition(tableDefinition, "(${LAST_WRITE.name} > ${LAST_PROPAGATE.name})")
+                    .name(prefix + "_last_propagate_idx")
```

Comment:
+ _last_transport_idx ?

Reply:
No, this is the existing index on lastPropagate